src/index.js

```diff
@@ -1,9 +1,22 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
-import App from './components/App.js.jsx';
-import './css/index.css';
+import React from 'react'
+import ReactDOM from 'react-dom'
+import { Router, Route, browserHistory } from 'react-router'
+import App from './components/App.js.jsx'
+import BuilderContainer from './components/containers/BuilderContainer.js.jsx'
+import DashboardContainer from './components/containers/DashboardContainer.js.jsx'
+import NewDay from './components/UI/NewDay.js.jsx'
+import SavedDays from './components/UI/SavedDays.js.jsx'
+import './css/index.css'
 
 ReactDOM.render(
-  <App />,
+  <Router history={ browserHistory }>
+    <Route path="/" component={ App }>
+      <Route path="/my-account" component={ DashboardContainer }>
```

Comment:
I think `my-account` is a good design decision because if you user the `:userName` people will be able to see other people's account page unless you have specific security measures in place restricting that. With `my-account` your architecture is set up to only go to the account page of who logged in. This is based on the assumption you don't want regular users to see another user's account.